include/gnc/constants.hpp

```diff
@@ -76,7 +76,7 @@ GNC_TRACKED_CONSTANT(constexpr static float, J_wheel, 135.0e-7f);
 
 GNC_TRACKED_CONSTANT(constexpr static float, w_wheel_max, 677.0f);
 
-GNC_TRACKED_CONSTANT(constexpr static uint64_t, NANOSECONDS_IN_WEEK, 7ULL*24ULL*60ULL*60ULL*1'000'000'000ULL);
+GNC_TRACKED_CONSTANT(constexpr static int64_t, NANOSECONDS_IN_WEEK, 7ULL*24ULL*60ULL*60ULL*1'000'000'000ULL);
```

Comment:
I'm assuming the shift in data type is because of python's handling of integers? Is this a concern for flight that'd we'd overflow the signed value?

Reply:
I was doing a lot of casting to int64_t for subtractions, and the Matlab python compatibility thing. There shouldn't be an overflow concern because 2^63 ns is about 300 years, I also added `MAXGPSTIME_NS` and `MINGPSTIME_NS`, where Orbits initialized out of that range are invalidated, so hopefully nothing weird can happen.